packages/components/pie-button/src/index.ts

```diff
@@ -226,18 +226,22 @@ export class PieButton extends FormControlMixin(LitElement) implements ButtonPro
             responsiveSize,
         } = this;
 
+        const classes = {
+            'o-btn': true,
+            'o-btn--fullWidth': isFullWidth,
+            'is-responsive': isResponsive,
```

Comment:
I'm a little skeptical of the `is-responsive` class name. Is this a convention we want to establish versus utility class names?

Reply:
I don't mind really – would you expect it to be something like `o-btn--responsive` instead?

Reply:
I think so yeah, unless its we make it a util class local to this component.